### PR TITLE
Vcon 52 header footer

### PR DESCRIPTION
**Pull request by shewood**
[vcon-52](https://decdvirtualconcierge.atlassian.net/jira/software/projects/VCON/boards/2?selectedIssue=VCON-52)

Added header bar, logo and language toggle (although it doesn't toggle yet)
added footer with canada logo
Added a date that is just set to the current date and time for now (will need to be updated to set the build date)

### CODE REVIEW

File: src/components/molecules/TheHeader.vue [hunk 1]
```diff
@@ -1,33 +1,120 @@
...
+                      stroke-linecap="round"
+                      stroke-linejoin="round"
+                      stroke-width="2"
+                      d="M21 21l-6-6m2-5a7 7 0 11-14 0 7 7 0 0114 0z"
+                    />
+                  </svg>
+                </span>
+              </button>
+            </form>
+          </div>
```
Comment (DaceyTom2) [inline src/components/molecules/TheHeader.vue:106]: The search bar can be hidden in mobile

**Review by DaceyTom2 — COMMENTED** [review]
I believe the little menu with an arrow is part of the header. I don't see a separate task for it or the "Service Canada Labs" banner

File: src/components/molecules/TheHeader.vue (hunk 1 above)
Comment (shewood) [reply src/components/molecules/TheHeader.vue:106]: I asked Thomas about the hidden or not hidden on mobile and I mentioned that I had it already showing on mobile (it was one of my questions) and he said to leave it showing on mobile.

Comment (DaceyTom2) [reply src/components/molecules/TheHeader.vue:106]: Sounds good!

**Review by TishamIslam — APPROVED** [review]
Looks good, there's one other little change down below, but aside from that you can go ahead and merge.

**Pull request merged**